app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,93 @@
+class PaymentsController < ApplicationController
+  require 'hips'
+
+  protect_from_forgery except: :webhook
+
+  def create
+    payment_type = params[:type]
+    user_id = params[:user_id]
+
+    @payment = Payment.create(payment_type: payment_type,
+                              user_id: user_id,
+                              status: Payment.order_to_payment_status(nil))
+
+    urls = { success: payment_success_url(user_id: user_id, id: @payment.id) }
+    urls[:error]   = payment_error_url(user_id: user_id, id: @payment.id)
+
+    urls[:webhook] = (SHF_WEBHOOK_HOST || root_url) +
```

Comment:
I think this should be a separate method.  It'd be a simple method, obviously, but refactoring it would provide the opportunity to express intent via the method name.  Ex:
```
urls[:webhook] = construct_webhook
```
and then
```
def construct_webhook
  (SHF_WEBHOOK_HOST || root_url) +   payment_webhook_path.sub('/en', '')
end
```
Also -- Is it possible that we'd also need to check (and remove) "/sv" from the URL?  (Perhaps this is taken care of in how we handle locale already.)

Reply:
Yes, good suggestion.

We don't need to filter `sv` as the routing-filter gem is configured to _not_ include the default locale.